### ISSUE TITLE
If contain any alphabet letter in between it still returns true

### ISSUE THREAD

**Issue by miguel550** [developersdo/sdq#1]
import sdq from 'sdq'

sdq.isCedula('40222075711')  == sdq.isCedula('402aaaaaaaaaaaa22075711')

In the case of having alphabet letter it should return false.

**Comment by rmariuzzo** [comment]
Good catch, the implementation removes everything except the digits. It should definitively behave differently. Thanks for reporting this.

**Comment by gabbanaesteban** [comment]
Fixed in #2
